qutebrowser/browser/commands.py

```diff
@@ -282,7 +282,12 @@ def tab_pin(self, count=None):
         to_pin = not tab.data.pinned
         tab_index = self._current_index() if count is None else count - 1
         cmdutils.check_overflow(tab_index + 1, 'int')
-        self._tabbed_browser.set_tab_pinned(tab_index, to_pin)
+        tab = self._cntwidget(count)
+
+        if tab is None:
+            raise cmdexc.CommandError("Unable to find tab '{}'.".format(count))
```

Comment:
Not because of the `check_overflow` - that only makes sure the index isn't too big for a C++ `int`, which caused exceptions in earlier PyQt versions. However, `check_overflow` is already done in `_cntwidget`, so it's actually pointless here.

However, at the top of the function `tab is None` is already checked.